### DISCUSSION TITLE
parameters for active subnetowrk search

### DISCUSSION THREAD

**Discussion by safrikDut** [egeulgen/pathfindR#92] [General]
Hi!
Thanks a lot for PathifindR, I am using it a lot for my project as I have to deal with two RNA-seq datasets. I am having a hard time trying to understand what are the parameters "score quantile threshold" and "significant gene threshold". I mean, I have read the description several times and in the paper from the package is also not really explicit for non-statiticians. Could someone explain them in a easier/biological way?

Thanks a lot for your help!!!

**Comment by egeulgen** [comment]
Hey @safrikDut,

Those two parameters are for filtering the active subsetwork search results, i.e. the list of active subnetworks, each of which is assigned a score. The "score quantile threshold" filters the list keeping the subnetworks with score over the given quantile (default = 0.8, set to -1 for disabling this). The "significant gene threshold" is the threshold for the minimum proportion of significant (input) genes in a given subnetwork (Default = 0.02), e.g., if the threshold is 0.02, the number of genes in a subnetwork is 100, and the number of input genes within the subnetwork is 1, the subnetwork is discarded (minimum required is 100 * 0.02 = 2)

Hope this helps,
Best,
-E

**Reply by safrikDut** [reply]
Thanks a lot for your prompt reply! If I may ask one more question: what would be the difference in the pathway enrichment resulting when choosing score quantile 0.8 or disabling -1? I am sorry to be bugging too much but my supervisor made this question and I still cannot answer it

Best,
Angeles

**Reply by egeulgen** [reply]
You will be using the low-scoring subnetworks as well, which may lead to more false-positive enrichment resultts
